src/client/components/navigation-button/navigation-button.style.css

```
@@ -1,21 +1,21 @@
 .icon {
+  color: var(--dark-grey);
   font-size: 2.1875rem;
-  color: rgb(94, 97, 99);
   margin-top: 0.2em;
 }
 p {
-  color: rgb(103, 107, 110);
+  color: var(--dark-grey);
```

it should be also var(--dark-blue)
```suggestion
  color: var(--dark-blue);
```